TestON/drivers/common/cli/emulator/remotemininetdriver.py

```diff
@@ -376,9 +376,20 @@ def disconnect( self ):
         if self.handle:
             # Close the ssh connection
             self.handle.sendline( "" )
-            self.handle.expect( "\$" )
-            self.handle.sendline( "exit" )
-            self.handle.expect( "closed" )
+            #self.handle.expect( "\$" )
+            i = self.handle.expect( [ '\$', 'mininet>', pexpect.TIMEOUT ],
```

Comment:
Ok. So you are starting a new mininet topology by running a .py file. Then you should be using another regular Mininet handle with startNet to run this .py file. Just trying to keep this remotemininet driver to follow what it is intended for and not to mix regular mininet handle mechanism when you can create one more local and exit properly.